pallets/file-system/src/lib.rs

```diff
@@ -126,6 +133,25 @@ pub mod pallet {
             + FixedPointNumber
             + EnsureFrom<u128>;
 
+        /// The currency mechanism, used for paying for reserves.
+        type Currency: ReservableCurrency<Self::AccountId>;
```

Comment:
I think this was being phased out for fungible::hold ([issue](https://github.com/paritytech/polkadot-sdk/issues/226))

Reply:
Thanks for spotting this. I changed the trait to `Currency` instead since I don't need any specific functionality.